packages/core/src/mesh/SkinnedMeshRenderer.ts

```
@@ -263,10 +256,14 @@ export class SkinnedMeshRenderer extends MeshRenderer {
         } else {
           shaderData.disableMacro("RENDERER_HAS_SKIN");
         }
+        const skin = this._skin;
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)